edison/static/js/policy.controller.js

```diff
@@ -10,6 +10,34 @@ function capitalize(string) {
     return `${string.charAt(0).toUpperCase()}${string.slice(1)}`
 }
 
+function commandField(policy)
+{
+    var commandRow = createInitElement('div', 'form-row')
+    var light =   createCommandField('light', policy)
+    var ac =   createCommandField('ac', policy);
+    var shutters =  createCommandField('shutters', policy);
+    
+    commandRow.appendChild(light);
+    commandRow.appendChild(ac);
+    commandRow.appendChild(shutters);
+    return commandRow;
+}
+
+
+function createCommandField(fieldName, policy) {
+
```

Comment:
just for the readability of this function, and the function above (commandField), I recommend to change the names.
maybe call this one "createCommandFieldParameters", and the one above should be called "createCommandField".
in this way, it's more clear to understand what the function is doing

Reply:
OK